frontend/src/components/UserProfile.jsx

```diff
@@ -1,12 +1,70 @@
 import "./UserProfile.css";
+import { useContext } from "react";
+import {UserContext} from './InfoProvider';
+
+// const [editProfile, setEditProfile] = useContext()
 
 function UserProfile() {
+
+   const [isLog, setIsLog] = useContext(UserContext)
+  
+
+  const ManageProfileHandler = (e) =>{
+    return console.log( "hello im from edit profile btn")
+  }
```

Comment:
In the code above the spaces are not really consistent, it would be best to update this. Also, function names should start with a lowercase character